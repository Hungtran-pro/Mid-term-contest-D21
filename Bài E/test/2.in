10
11 1
???aa?a??aa
15 2
b?b?babbb??aa?a
19 3
c?b?a???aa???caac?a
16 4
aadd?bbbdadbaccb
10 5
dc?deeadad
18 1
??????????????????
18 2
?????????aaaaaaaaa
14 3
ccccbbbbaaaa??
16 4
aaaa????????????
20 5
cccc????eeeebbbb????
