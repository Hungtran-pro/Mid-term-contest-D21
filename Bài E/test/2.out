11
5
3
2
1
18
9
4
4
4
